doc/whats_new/v1.4.rst

```diff
@@ -263,11 +270,20 @@ Changelog
   missing indices twice to improve efficiency.
   :pr:`27017` by :user:`Xuefeng Xu <xuefeng-xu>`.
 
-- |MajorFeature| :class:`preprocessing.MinMaxScaler` and :class:`preprocessing.MaxAbsScaler` now
+- |Enhancement| Improves warnings in :class:`preprocessing.FunctionTransfomer` when
+  `func` returns a pandas dataframe and the output is configured to be pandas.
+  :pr:`26944` by `Thomas Fan`_.
+
+- |Enhancement| :class:`preprocessing.TargetEncoder` now supports `target_type`
+  'multiclass'. :pr:`26674` by :user:`Lucy Liu <lucyleeow>`.
+
+- |MajorFeature| :class:`preprocessing.MinMaxScaler` and
```

Comment:
We should respect the standard label ordering MajorFeature > Feature > ... > Enhancement > Fix.